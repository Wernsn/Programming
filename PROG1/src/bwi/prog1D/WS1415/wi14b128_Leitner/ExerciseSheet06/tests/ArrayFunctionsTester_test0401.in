4
33
4
30
4
39
4
12
0