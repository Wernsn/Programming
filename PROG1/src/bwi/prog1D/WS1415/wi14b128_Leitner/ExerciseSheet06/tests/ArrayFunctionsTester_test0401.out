find 1: 3 (33)
find 1: 0 (30)
find 1: 9 (39)
find 1: -1 (12)
